gradio/cli/cli.py

```diff
@@ -36,5 +43,7 @@ def cli():
         except ValueError:
             console = Console()
             console.print_exception()
+    elif args[0] in {"hf-upload-mcp"}:
```

Comment:
I think we shoudl rename `hf-upload-mcp` to just `upload-mcp` because it doesn't make sense if the gradio app is running locally or hosted on a server other than hf spaces.

Reply:
Yea agreed